packages/paste-website/src/pages/api/ai.ts

```diff
@@ -19,8 +19,12 @@ import type {NextApiRequest, NextApiResponse} from 'next';
 export default async function handler(req: NextApiRequest, res: NextApiResponse): Promise<void> {
   const {question, secret} = req.body;
   // Exit early if the required params aren't provided
-  if (!question || secret !== process.env.OPENAI_API_SECRET) {
-    res.status(200).send({answer: 'Please provide a question'});
+  if (secret !== process.env.OPENAI_API_SECRET) {
```

Comment:
better http status code usage and splitting up these errors so it's more clear what is failing during testing